alacritty/src/event.rs

```diff
@@ -758,6 +759,15 @@ impl<N: Notify + OnResize> Processor<N> {
         processor.ctx.terminal.dirty = true;
     }
 
+    fn start_program(program: &Program) {
+        let args = program.args().to_vec();
+
+        match start_daemon(program.program(), &args) {
+            Ok(_) => debug!("Launched {} with args {:?}", program.program(), args),
+            Err(_) => warn!("Unable to launch {} with args {:?}", program.program(), args),
+        }
```

Comment:
We have this construct in a lot of places, should it maybe be moved to `start_daemon`?

If it's used just for visual bell, it probably shouldn't be `start_program`, but I see nothing particularly visual bell specific here.

Reply:
We can't abstract, since messages are different everywhere, so you may move, but you should supply the messages, so I feel like it's fine the way it is now.

Reply:
Do the messages need to be different? Or would we be fine with just having a uniform message for launching processes and having that  fail?

I feel like it's probably not super necessary to be able to have a custom message for every different way we want to spawn a program?